packages/integrations/src/awslambda.ts

```diff
@@ -0,0 +1,255 @@
+      const remainingTimeInMillisecond: number = this._awsContext.getRemainingTimeInMillis();
+      const executionTime: number = configuredTimeInMilliseconds - remainingTimeInMillisecond;
+
+      scope.setContext('lambda', {
+        aws_request_id: this._awsContext.awsRequestId,
+        function_name: this._awsContext.functionName,
+        function_version: this._awsContext.functionVersion,
+        invoked_function_arn: this._awsContext.invokedFunctionArn,
+        execution_duration_in_millis: executionTime,
+        remaining_time_in_millis: this._awsContext.getRemainingTimeInMillis(),
+      });
+    };
+
+    /**
+     * This function use to generate cloud watch url
+     */
+    const cloudwatchUrl = (): string => {
+      /**
+       * process.env.AWS_REGION -  this parameters given the AWS region
+       * process.env.AWS_LAMBDA_FUNCTION_NAME - this parameter provides the AWS Lambda Function name
+       */
+      return `https://${process.env.AWS_REGION}.console.aws.amazon.com/cloudwatch/home?region=${process.env.AWS_REGION}#logsV2:log-groups/log-group/$252Faws$252Flambda$252F${process.env.AWS_LAMBDA_FUNCTION_NAME}`;
+    };
+
+    /**
+     * This function sets Cloud Watch Logs data which are displayed in Sentry Dashboard
+     * @param scope  - holds additional event information
+     * @hidden
+     */
+    const setCloudwatchLogsData = (scope: Scope): void => {
+      scope.setContext('cloudwatch logs', {
+        log_group: this._awsContext.logGroupName,
+        log_stream: this._awsContext.logStreamName,
+        url: cloudwatchUrl(),
+      });
+    };
+
+    /**
+     * This function sets tags which are displayed in Sentry Dashboard
+     * @param scope  - holds additional event information
+     * @hidden
+     */
+
+    const setTags = (scope: Scope): void => {
+      scope.setTag('runtime', `node${global.process.version}`);
+      scope.setTag('transaction', this._awsContext.functionName);
+      scope.setTag('runtime.name', 'node');
+      scope.setTag('server_name', process.env._AWS_XRAY_DAEMON_ADDRESS || '');
+      scope.setTag('url', `awslambda:///${this._awsContext.functionName}`);
+    };
+
+    const flushTime = this._flushTime;
+    // timeout warning buffer for timeout error
+    const timeoutWarningBuffer: number = 1500;
+
+    const configuredTimeInSec = Math.floor(configuredTimeInMilliseconds / 1000);
+    const configuredTimeInMilli = configuredTimeInSec * 1000;
+
+    /**
+     * This function is invoked when there is timeout error
+     * Here, we make sure the error has been captured by Sentry Dashboard
+     * and then re-raise the exception
+     * @param configuredTime  - configured time in seconds
+     * @hidden
+     */
+    const timeOutError = (configuredTime: number): void => {
+      setTimeout(() => {
+        const error = new Error(
+          `WARNING : Function is expected to get timed out. Configured timeout duration = ${configuredTimeInSec +
```

Comment:
You pass one timeout as an argument, and one is taken from the upper-scope. This whole thing can be just moved to line 213 without an additional function that basically serves no purpose here.

Reply:
Fixed in the latest Pull request.